platforms/core-runtime/wrapper-main/src/integTest/groovy/org/gradle/integtests/WrapperSupportedBuildJvmIntegrationTest.groovy

```diff
@@ -40,19 +40,31 @@ class WrapperSupportedBuildJvmIntegrationTest extends AbstractWrapperIntegration
         failure.assertHasErrorOutput("Gradle requires JVM 8 or later to run. You are currently using JVM ${jdk.javaVersionMajor}.")
 
         where:
-        jdk << AvailableJavaHomes.getJdks("1.6", "1.7")
+        jdk << AvailableJavaHomes.getUnsupportedWrapperJdks()
     }
 
-    def "provides reasonable failure message when attempting to run build under java #jdk.javaVersion"() {
```

Comment:
This test did not actually test the wrapper using a different java version, but tests running the daemon with a different java version. 

Running the daemon with different java version is covered by `SupportedBuildJvmVersionIntegrationTest`